nginx/Brewfile.lock.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
I don't think we want this?!

Reply:
Should I delete both lock files, or just the Brewfile.lock file?

Reply:
updating the yarn.lock isn't a bad thing, I think it's fairly safe to leave in

Reply:
yarn.lock should be kept in
> All yarn.lock files should be checked into source control. This allows Yarn to install the same exact dependency tree across all machines, whether it be your coworker’s laptop or a CI server.
(from [yarn documentation](https://classic.yarnpkg.com/en/docs/yarn-lock/))

Don't know about the brew one though

Reply:
Brewfile lock deleted. Yarn lock kept